src/toil/test/wdl/wdltoil_test.py

```diff
@@ -90,6 +114,17 @@ def check(self, p: subprocess.CompletedProcess) -> None:
 
         p.check_returncode()
 
+    @slow
+    def test_unit_tests_v11(self):
+        # There are still some bugs with the WDL spec, use a fixed version until
```

Comment:
This ought to indicate which issues need to be fixed before we can swap back to the official spec.

Reply:
I linked the relevant PR on openwdl